### DISCUSSION TITLE
对话中总token数大于约1500时返回403错误

### DISCUSSION THREAD

**Discussion by LarsPh** [lobehub/lobehub#4263] [General Question] [answered]
在使用docker服务器私有部署的lobechat时，总token超过大约1500时返回
Sorry, the server has rejected your request. You do not have permission to access this content.
{
  "type": 403,
  "message": "Sorry, the server has rejected your request. You do not have permission to access this content."
}
试了不同的API和模型（gpt-4o, claude 3.5 sonnet）都是一样的结果。一开始没配置DEFAULT_AGENT_CONFIG，后面试着配置了
DEFAULT_AGENT_CONFIG=chatConfig.enableHistoryCount=true;chatConfig.historyCount=300;params.max_tokens=16384
但问题还是存在。麻烦开发者帮忙看一下🙏

**Comment by arvinxx** [comment]
这个问题我没有遇到过，你的key是官方正版的吗？有可能是你的key 供应商的问题。
如果key正版情况下，试下本地部署是否有这个问题？如果本地部署没问题的话建议检查下nginx相关的配置。如果本地部署有问题的话把你的query贴下，我来尝试复现

**Comment by LarsPh** [comment, marked as answer]
应该和lobechat, API key无关，因为我用的amazon cloudfront默认防火墙规则里面限制body size不大于8kb...改掉就解决了。感谢指导！
